CMakeLists.txt

```diff
@@ -37,7 +29,7 @@ option( EXIV2_BUILD_DOC               "Add 'doc' target to generate documentatio
 
 # Only intended to be used by Exiv2 developers/contributors
 option( EXIV2_TEAM_EXTRA_WARNINGS     "Add more sanity checks using compiler flags"           OFF )
-option( EXIV2_TEAM_WARNINGS_AS_ERRORS "Treat warnings as errors"                              OFF )
+option( EXIV2_TEAM_WARNINGS_AS_ERRORS "Treat warnings as errors"                              ON )
```

Comment:
Please don't make this the default. We can activate it on the CI, but this is not a sensible default, as we only test a subset of the available compiler version, which report different warnings. This will make builds randomly fail for downstream projects when they upgrade their compiler.

This will also kill the gitlab builds, since they use `-DEXIV2_TEAM_EXTRA_WARNINGS=ON` and at least on Fedora, that does not even work in debug builds.

Reply:
Good point. I will change it.